src/Parser.js

```diff
@@ -129,17 +133,17 @@ export function parse (value, root, parent, rule, rules, rulesets, pseudo, point
 
 /**
  * @param {string} value
- * @param {Element} root
- * @param {Element?} parent
+ * @param {Node?} root
+ * @param {Node?} parent
  * @param {number} index
  * @param {number} offset
  * @param {string[]} rules
  * @param {number[]} points
  * @param {string} type
```

Comment:
potentially we could explore limiting this to a particular union of strings and define the whole function in a similar fashion as [`node`](https://github.com/thysultan/stylis.js/blob/ceb1b0416fab3b4c451334b35c64a62529ab91dd/src/Tokenizer.js#L12-L23)